src/app/profile-page/profile-page-security-form/profile-page-security-form.component.ts

```diff
@@ -60,6 +60,12 @@ export class ProfilePageSecurityFormComponent implements OnInit {
    */
   @Input()
   FORM_PREFIX: string;
+
+  /**
+   * monitor to password is weak or not from server response
+   */
+  @Input()
+  isRobustPasswordError: boolean;
```

Comment:
It seems odd to me that you added a new `isRobustPasswordError` flag without updating/correcting the existing `isInValid` flag just above this on line 25.  This means it's currently possible for a password to seem "valid" but then turn out to be invalid (as "isRobustPasswordError" returns false).

I'd rather we find a way to merge these concepts. The UI shouldn't have two different definitions of validity for a password.  If you look closely at the `profile-page-security-form.component.ts` class, it has separate validation for validity (to check if the password is >6 chars) which do not correspond to the robust password definitions.